modules/tinymce/src/themes/silver/main/ts/ui/sizeinput/SizeInputToolbar.ts

```diff
@@ -0,0 +1,223 @@
+import {
+  AddEventsBehaviour, AlloyComponent, AlloyEvents,
+  FormCoupledInputs as AlloyFormCoupledInputs,
+  FormField as AlloyFormField,
+  Input as AlloyInput,
+  AlloySpec, AlloyTriggers, Behaviour, CustomEvent, Disabling,
+  Focusing,
+  FocusInsideModes,
+  GuiFactory,
+  Keying,
+  NativeEvents, Representing, SketchSpec, Tabstopping, Tooltipping
+} from '@ephox/alloy';
+import { Cell, Fun, Id, Optional, Optionals, Unicode } from '@ephox/katamari';
+import { Focus, SelectorFind, SugarElement } from '@ephox/sugar';
+
+import { formInputEvent } from 'tinymce/themes/silver/ui/general/FormEvents';
+
+import { UiFactoryBackstageProviders } from '../../backstage/Backstage';
+import * as UiState from '../../UiState';
+import { onControlAttached, onControlDetached } from '../controls/Controls';
+import * as Icons from '../icons/Icons';
+import { formatSize, makeRatioConverter, noSizeConversion, parseSize, SizeConversion } from './SizeInputModel';
+
+interface RatioEvent extends CustomEvent {
+  readonly isField1: boolean;
+}
+
+export interface SizeInputGenericSpec<ApiType = never> {
+  readonly label: Optional<string>;
+  readonly enabled: boolean;
+  readonly width: string;
+  readonly height: string;
+  readonly onEnter: Optional<(input: AlloyComponent) => Optional<boolean>>;
+  readonly onInput: Optional<(input: AlloyComponent) => void>;
+  readonly onSetup: Optional<(api: ApiType) => (api: ApiType) => void>;
+  readonly getApi: Optional<(input: AlloyComponent) => ApiType>;
```

Comment:
We will need to clean the SizeInput.ts from all this stuff that is only toolbar related also

Reply:
ok, now it should be closest as possible as the situation that we had before [this PR](https://github.com/tinymce/tinymce/pull/9919/files#diff-a548923930120ccb6068993232ce2bb394710c654ed84324be1c89ef8bb6c6fe), I left the disable in the context toolbar even if it is always false, since we have a Jira that talks about that possibility and maybe in the nearby future it could be changed.

Also I renamed the second `size-input-events` in `size-input-events2` as it was before, I don't think we have any conflict but it would be more safe